services/keepassService.js

```diff
@@ -10,65 +10,52 @@ let Case = require('case'),
 import { argon2 } from '$lib/argon2.js'
 import { parseUrl, getValidTokens } from '$lib/utils.js'
 
-function KeepassService(keepassHeader, passwordFileStoreRegistry, keepassReference) {
+function KeepassService(passwordFileStoreRegistry, keepassReference) {
 	var my = {};
 
-	var littleEndian = (function () {
-		var buffer = new ArrayBuffer(2);
-		new DataView(buffer).setInt16(0, 256, true);
-		return new Int16Array(buffer)[0] === 256;
-	})();
-
 	/** 
 	 * return Promise(arrayBufer)
 	 */
 	my.getChosenDatabaseFile = function (providerKey) {
 		return passwordFileStoreRegistry.getChosenDatabaseFile(providerKey)
 	}
 
-	my.getMasterKey = function (bufferPromise, masterPassword, keyFileInfo) {
+	my.getMasterKey = function (masterPassword, keyFileInfo) {
+		console.log(masterPassword || "none", keyFileInfo)
```

Comment:
Are you sure you wanted to commit that? 😂

Reply:
Haha!  Yeah this code is full of stuff like this.  It will get cleaned up at the end :P